src/Sentry/Internal/DebugStackTrace.cs

```diff
@@ -489,6 +496,13 @@ private static void DemangleLambdaReturnType(SentryStackFrame frame)
         }
     }
 
+#if TRIMMABLE
```

Comment:
how about having a single method declaration and only ifdef the body?

Reply:
I did try that initially... since the ifdef wraps another ifdef, it wasn't super readable, tbh. Also, this version will result in ever so slightly less code in an AOT app. 

I'm happy to go either way though if we'd prefer putting the ifdef in the body.

Reply:
Actually, not sure what I was talking about wrt the other ifdefs. Have just updated 👍🏻

Reply:
Have added a separate issue to refactor the `DebugStackTrace` class: https://github.com/getsentry/sentry-dotnet/issues/2761